### PR TITLE
Changing to use symbol instead of searchable so that a fuzzy search is n...

### LINKED ISSUE #851: Deposited files counts are wrong when using email for user_key
When using email as the `authentication_keys` with Devise (sufia v. 4.3.1) the deposited file and collection counts are wrong for me. The counts returned for an email `abc@bcd.net` are for all users with the domain name `bcd.net` their email. The methods returning bad counts are:
```
Sufia::DashboardHelperBehavior#number_of_files
Sufia::DashboardHelperBehavior#number_of_collections
Sufia::SufiaHelperBehavior#number_of_deposits
```

The solr query generated for `Sufia::DashboardHelperBehavior#number_of_files` is:
```
select?wt=ruby&rows=0&q=depositor_tesim:abc\@bcd\.net&qt=standard
```

I found that the above works if the search term is double quoted or the following query works:
```
select?wt=ruby&rows=0&q=depositor_ssim:abc\@bcd\.net&qt=standard
```

I think, I fixed it by changing `(Solrizer.solr_name('depositor', :stored_searchable)` to `Solrizer.solr_name('depositor', :symbol)`.

Additionally the catalog search linked from Profile->Deposited Files counts doesn't return result but that's probably a separate ticket?

### PR DESCRIPTION
**Pull request by cam156**
...ot used for the depositor

fixes #851

### CODE REVIEW

**Comment by awead** [comment]
@cam156 I restarted the build. Looks :+1: to me once Travis passes.

File: spec/helpers/dashboard_helper_spec.rb [hunk 1]
```diff
@@ -49,4 +49,60 @@
     end
   end
 
+
+  describe "#number_of_files" do
+    let(:conn) { ActiveFedora::SolrService.instance.conn }
+    let(:user1) { User.new(email:"abc@test") }
```
Comment (jcoyne) [inline spec/helpers/dashboard_helper_spec.rb]: Picking nits here, can you add a space after the colins? https://github.com/bbatsov/ruby-style-guide#spaces-operators

**Comment by phebal** [comment]
I think `Sufia::SufiaHelperBehavior#number_of_deposits` might need this adjustment as well. Thanks for the fix.

**Comment by cam156** [comment]
@phebal Added the other location... 
@jcoyne Added the spaces...

File: spec/helpers/sufia_helper_spec.rb [hunk 2]
```diff
@@ -164,4 +164,43 @@
...
+            Solrizer.solr_name('depositor', :symbol) => user1.user_key
+      end
+
+      # deposited by the second user, but editable by the first
+      conn.add  id: "1994", Solrizer.solr_name('depositor', :stored_searchable) => user2.user_key, "has_model_ssim"=>[model],
+          Solrizer.solr_name('depositor', :symbol) => user2.user_key, "edit_access_person_ssim" =>user1.user_key
+      conn.commit
+    end
+
+    def destroy_models
```
Comment (jcoyne) [inline spec/helpers/sufia_helper_spec.rb]: This is unnecessary, ActiveFedora::Cleaner will handle this.

File: spec/helpers/dashboard_helper_spec.rb [hunk 3]
```diff
@@ -49,4 +49,60 @@
...
+          Solrizer.solr_name('depositor', :symbol) => user1.user_key
+    end
+
+    # deposited by the second user, but editable by the first
+    conn.add  id: "1994", Solrizer.solr_name('depositor', :stored_searchable) => user2.user_key, "has_model_ssim"=>[model],
+        Solrizer.solr_name('depositor', :symbol) => user2.user_key, "edit_access_person_ssim" =>user1.user_key
+    conn.commit
+  end
+
+  def destroy_models
```
Comment (jcoyne) [inline spec/helpers/dashboard_helper_spec.rb]: This is unnecessary, ActiveFedora::Cleaner will handle this.

**Pull request merged**